catboost/libs/fstr/shap_values.h

```diff
@@ -71,16 +73,21 @@ void CalcShapValuesForDocumentMulti(
     int flatFeatureCount,
     TConstArrayRef<NCB::NModelEvaluation::TCalcerIndexType> docIndexes,
     size_t documentIdx,
-    TVector<TVector<double>>* shapValues
+    TVector<TVector<double>>* shapValues,
+    ECalcTypeShapValues calcType
```

Comment:
Please, add default value to avoid breaking change in interface here and in others functions too.

Reply:
These functions didn't have any default values previously. Wouldn't it be better to fix corresponding tests or so?

Reply:
Either way, added default calc type - normal